src/bpfmap.h

```diff
@@ -33,16 +33,14 @@ class BpfMapError : public ErrorInfo<BpfMapError> {
   }
 };
 
-using KeyType = std::vector<uint8_t>;
-using KeyVec = std::vector<KeyType>;
-using ValueType = std::vector<uint8_t>;
-using MapElements = std::vector<std::pair<KeyType, ValueType>>;
+using util::OpaqueValue;
+using MapElements = std::vector<std::pair<OpaqueValue, OpaqueValue>>;
 using BucketUnit = uint64_t;
-using BucketType = std::vector<BucketUnit>;
-using HistogramMap = std::map<KeyType, BucketType>;
+using BucketType = std::vector<uint64_t>;
```

Comment:
I'd either keep `BucketUnit` here or drop it completely.

Reply:
Dropped.